### PR TITLE
Update main.ts

### PR DESCRIPTION
**Pull request by lslzl3000**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -7,10 +7,9 @@ declare global {
     }
 }
 
-let a:any = 'test'
+let a:string = 'test'
```
Comment (lslzl3000) [inline src/main.ts:10]: test review

Comment (lslzl3000) [reply src/main.ts:10]: hi

**Review by lslzl3000 — COMMENTED** [review]
just test

File: src/main.ts (hunk 1 above)
Comment (SirXuNian) [reply src/main.ts:10]: 111

**Pull request closed without merging**